practice-one/src/index.html

```diff
@@ -41,37 +41,82 @@
-      <span class="banner__sub-title">RESPONSIVE UI KIT</span>
-      <div class="banner__content">
-        <h1 class="text-font">HOFMANN</h1>
-        <p class="banner__content--title">We’re Bring It To The New Level</p>
-        <p class="text-normal banner__content--decristion">
-          Every new computer that’s brought home from the
-          store has an operating system installed onto it.
-          But what most new computer users don’t realize,
-          is that without an operating system, that
-          computer would be a simple shell of
-          possibilities.
-        </p>
+    <!--START BANNER-->
+    <section class="banner-section">
+      <div class="banner__section--decristion">
+        <span class="banner__sub-title">RESPONSIVE UI KIT</span>
+        <div class="banner__content">
+          <h1 class="text-font">HOFMANN</h1>
+          <p class="banner__content--title">We’re Bring It To The New Level</p>
+          <p class="text-normal banner__content--decristion">
+            Every new computer that’s brought home from the
+            store has an operating system installed onto it.
+            But what most new computer users don’t realize,
+            is that without an operating system, that
+            computer would be a simple shell of
+            possibilities.
+          </p>
+        </div>
+        <div class="banner__section-app-img">
+          <img src="/practice-one/src/assets/images/appstore-badge.png" alt="app store" class="app-store-img" />
+          <img src="/practice-one/src/assets/images/google-play-badge.png" alt="google play" class="google-img" />
+        </div>
       </div>
-      <div class="banner__section-app-img">
-        <img src="/practice-one/src/assets/images/appstore-badge.png" alt="app store" class="app-store-img" />
-        <img src="/practice-one/src/assets/images/google-play-badge.png" alt="google play" class="google-img" />
+      <div class="banner__section-iphone">
+        <img src="/practice-one/src/assets/images/iphone-app.png" alt="iphone app">
       </div>
-    </div>
-    <div class="banner__section-iphone">
-      <img src="/practice-one/src/assets/images/iphone-app.png" alt="iphone app">
-    </div>
-  </section>
-  <!--END BANNER SECTION-->
+    </section>
+    <!--END BANNER SECTION-->
   </header>
   <!--END HEADER-->
   <main class="container__main">
     <!--START ADVANTAGES-->
-    <section></section>
+    <section class="advantages__section">
+      <div class="advantages__section--header ">
+        <h2 class="title-h2 text-font advantages__header--title ">Stunning Results</h2>
+        <p class="advantages__header--description">Just look to the statistics we have generate the past year.</p>
+      </div>
+      <div class="advantages__section--content">
+        <div class="advanteages__main--content">
+          <div class="advanteages__main--content-img">
+            <img src="/practice-one/src/assets/images/circle-1.png" alt="circle 74,98">
```

Comment:
Can we use  css to custom progress bar. The percentage can be change, not static